en/micro-integrator/docs/setup/deployment/k8s-pipeline/deploy-your-own-artifact.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Your Own Artifact
+1.  Fork the [CICD sample](https://github.com/wso2-incubator/cicd-sample-docker-mi) repository which contains a [Maven
+    Multi-Module
+    project](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/create-multi-maven-project/).
```

Comment:
Change this to an integration project and point to the docs on this. With 7.1.0 an Integration Project can be created as an MMM project.